service/chat/auth.py

```diff
@@ -36,11 +36,11 @@
 """
 
 def deny():
-    sys.stdout.buffer.write(b'\x00\x02\x00\x00')
+    sys.stdout.buffer.write(b'\x00\x00\x00\x00')
     sys.stdout.flush()
 
 def allow():
-    sys.stdout.buffer.write(b'\x00\x02\x00\x01')
+    sys.stdout.buffer.write(b'\x00\x00\x00\x01')
```

Comment:
The [documentation](https://esl.github.io/MongooseDocs/latest/authentication-methods/external/) says the script should have returned `0x0000` and `0x0001` yet authentication seemed to work anyway. I'm making this change just to match the spec.